horizon/enforcer/api.py

```diff
@@ -62,17 +69,30 @@ def transform_headers(request: Request) -> dict:
     }
 
 
-def log_query_result(query: AuthorizationQuery, response: Response):
+def log_query_result(query: BaseSchema, response: Response):
     """
     formats a nice log to default logger with the results of permit.check()
     """
-    params = "({}, {}, {})".format(query.user.key, query.action, query.resource.type)
+    params = repr(query)
     try:
         result: dict = json.loads(response.body).get("result", {})
-        allowed = result.get("allow", False)
+        allowed: Dict | List[bool] = result.get("allow", None)
```

Comment:
The annotation should probably be bool | list[bool]

Reply:
fixed